src/views/Covid19/index.tsx

```diff
@@ -133,6 +135,7 @@ function Covid19(props: Props) {
     const [indicatorListPending, indicatorListResponse] = useRequest<MultiResponse<Indicator>>(
         indicatorListGetUrl,
     );
+    const indicatorList = indicatorListResponse?.results || emptyIndicatorList;
```

Comment:
You don't need to assign empty list as default.
You can simply do `indicatorList?.find(i => indicatorKeySelector(i) === selectedIndicator)`